src/providers/pinoLoggerOptionsBuilder.ts

```diff
@@ -115,7 +115,10 @@ export class PinoLoggerOptionsBuilder {
       function getMessageFromError(args) {
         const [o, ...n] = args as [Record<string, unknown>, ...unknown[]];
```

Comment:
what is o? What is n? 
go full naming

Reply:
`o` is message for log.
`n` is additional params from `logger` method